src/request-scope-storage.js

```diff
@@ -0,0 +1,61 @@
+const continuationStorage = require('continuation-local-storage')
```

Comment:
Shall we rename the file to `mdc.js` or `mapped-diagnostic-context.js`? You call it MDC from other files anyways...